File: fetch_test.py
import urllib.request
import urllib.error
import json

url = 'https://morentiumapp.vercel.app/api/verify_key'
data = json.dumps({'key': 'test', 'hwid': 'test'}).encode('utf-8')
req = urllib.request.Request(url, data=data, headers={'Content-Type': 'application/json'}, method='POST')

try:
    response = urllib.request.urlopen(req)
    print(response.read().decode('utf-8'))
except urllib.error.HTTPError as e:
    print(f"HTTP Error {e.code}:\n{e.read().decode('utf-8')}")
except Exception as e:
    print(f"Unknown Error: {e}")

File: fetch_test.js
const https = require('https');

function fetch(url) {
    https.get(url, (res) => {
        let data = '';
        res.on('data', (chunk) => { data += chunk; });
        res.on('end', () => {
            console.log(`\n--- Status: ${res.statusCode} [${url}] ---`);
            console.log(data);
        });
    }).on('error', (e) => {
        console.error(e);
    });
}

fetch("https://morentiumapp.vercel.app/api/check-subscription?userId=999");
fetch("https://morentiumapp.vercel.app/api/leaderboard");
